lib/demon/sidekiq.rb

```diff
@@ -70,6 +70,14 @@ def self.max_allowed_sidekiq_rss_bytes
     [ENV["UNICORN_SIDEKIQ_MAX_RSS"].to_i, DEFAULT_MAX_ALLOWED_SIDEKIQ_RSS_MEGABYTES].max.megabytes
   end
 
+  def stop_signal
+    "TERM"
+  end
+
+  def stop_timeout
+    30
```

Comment:
I don't think there is a need for us to wait 30 seconds here. Something short like 10 seconds is fine IMO since the failed job will be picked up again by Sidekiq when it restarts. This does mean we have to pass a shorter timeout to sidekiq via the `-t` CLI option.